specification/billing/resource-manager/Microsoft.Billing/preview/2018-11-01-preview/billing.json

```diff
@@ -3261,6 +3261,105 @@
           }
         }
       }
+    },
+    "/subscriptions/{subscriptionId}/providers/Microsoft.Billing/billingAccounts/default/lineOfCredit/default": {
+      "get": {
+        "tags": [
+          "LineOfCredits"
+        ],
+        "x-ms-examples": {
+          "LineOfCreditBySubscription": {
+            "$ref": "./examples/LineOfCreditBySubscription.json"
+          }
+        },
+        "operationId": "LineOfCredits_Get",
+        "description": "Get the current line of credit.",
+        "parameters": [
+          {
+            "$ref": "#/parameters/apiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/subscriptionIdParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK. The request has succeeded.",
+            "schema": {
+              "$ref": "#/definitions/LineOfCredit"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/providers/Microsoft.Billing/billingAccounts/default/lineOfCredit/default/increase": {
+      "post": {
+        "tags": [
+          "LineOfCredits"
+        ],
+        "operationId": "LineOfCredits_Increase",
+        "x-ms-long-running-operation": true,
+        "x-ms-examples": {
+          "IncreaseLineOfCreditBySubscription": {
+            "$ref": "./examples/IncreaseLineOfCreditBySubscription.json"
+          }
+        },
+        "description": "Increase the current line of credit.",
+        "parameters": [
+          {
+            "$ref": "#/parameters/apiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/subscriptionIdParameter"
+          },
+          {
+            "name": "parameters",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/IncreaseLineOfCreditRequestProperties"
+            },
+            "description": "Parameters supplied to the increase line of credit operation."
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK. The request has succeeded.",
+            "schema": {
+              "$ref": "#/definitions/LineOfCredit"
+            }
+          },
+          "202": {
+            "description": "Accepted. Line of credit increase is in progress.",
+            "headers": {
+              "Location": {
```

Comment:
@asarkar84 any reason you want to rely on Location header protocol. It's recommended you use Azure-AsyncOperation to track the status of your operation.

Reply:
True, Azure-AsyncOperation is the recommended way to poll the async operation. 
Once the Azure-AsyncOperation operation is completed that will return location in response. Some of our consumers use location property so we keep both. 
We follow the same pattern in our existing APIs for 202 status.